tests/testing.md

```diff
@@ -121,6 +121,15 @@ See
 ### Running Docker Compose Integration Tests
 
 1. Have Docker desktop running
+1. Copy the 'Voxel51 GitHub Legacy' license file to `docker/legacy-license.key`
+
+* gs://voxel51-test/licenses/299a423b/1/license.key
+
+1. Copy the 'Voxel51 GitHub Internal' license file to
+   `docker/internal-license.key`
+
+* gs://voxel51-test/licenses/299a423b/1/license.key
```

Comment:
Is the same license file from Google storage used as both

* `docker/legacy-license.key`
* `docker/internal-license.key`

?

Reply:
I mean, the tests pass, licenses don't actually matter right now other than they exist, but sure - let's let folks figure out how to get their own licenses.